docs/tutorials/training_to_inference.md

```diff
@@ -0,0 +1,201 @@
+that performs the conversion for our model.
+
+Going back to our example, we are using the `efficientnet-b0` and the `preprocessing` models, with the latter containing
+`dali.py` file defined above. To use these two together, we are creating the ensemble model `efficientnet_ensemble`. 
+Below is the outline of the model repository in our example. You can find the [model configuration](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/user_guide/model_configuration.html) 
+files in the [Efficientnet example](https://github.com/triton-inference-server/dali_backend/tree/main/docs/examples/efficientnet/model_repository) 
+in DALI Backend repository.
+
+```
+model_repository
+├── efficientnet-b0
+│   ├── 1
+│   │   └── model.plan
+│   └── config.pbtxt
+├── efficientnet_ensemble
+│   ├── 1
+│   └── config.pbtxt
+└── preprocessing
+    ├── 1
+    │   └── dali.py
+    └── config.pbtxt
+```
+
+Last step is the configuration of DALI model in the model repository. As mentioned earlier, you can use Triton's
+Model Autoconfiguration or provide explicit `config.pbtxt` file. According to the outline above, below is the example of
+simple configuration file for DALI model (`model_repository/preprocessing/config.pbtxt`). Please note that the name of
+the `input` matches the name of the `fn.external_source` operator provided in the `inference_pipe`.
+
+```
+name: "preprocessing"
+backend: "dali"
+max_batch_size: 32
+input [
+  {
+    name: "DALI_INPUT"
+    data_type: TYPE_UINT8
+    dims: [ -1 ]
+  }
+]
+
+output [
+  {
+    name: "DALI_OUTPUT"
+    data_type: TYPE_FP32
+    dims: [ 3, 224, 224 ]
+  }
+]
+```
+
+For the details about the model configuration files please refer to [Triton documentation](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/user_guide/model_configuration.html).
+
+### ➔ 3 : Run Triton server and send requests
+
+Please refer to [Triton documentation](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/index.html)
+for any details about running the server and leveraging Triton Client module to send requests to the server.
+
+### Triton Client hints
+
+As mentioned earlier, you can find the examples of Triton client implementation inside [DALI Backend repository](https://github.com/triton-inference-server/dali_backend/tree/main/qa). 
+The purpose of this tutorial is not to fully explain how to create a Triton client module. However, the following are 
+a couple of suggestions for creating a Triton client specifically for DALI model.
+
+#### Loading data as binary buffer
+NVIDIA hardware offers image and video decoding acceleration using [Hardware JPEG Decoder](https://developer.nvidia.com/blog/leveraging-hardware-jpeg-decoder-and-nvjpeg-on-a100/) and [NVDEC cores](https://developer.nvidia.com/video-codec-sdk). 
```

Comment:
I would make this broader. We have also nvJPEG2k and nvTIFF, which uses just SMs. So how about:
```suggestion
NVIDIA offers GPU accelerated image and video decoding using various software (i.e. [nvJPEG2k](https://docs.nvidia.com/cuda/nvjpeg2000/userguide.html) or [nvTIFF](https://docs.nvidia.com/cuda/nvtiff/)) and hardware (like [JPEG](https://developer.nvidia.com/blog/leveraging-hardware-jpeg-decoder-and-nvjpeg-on-a100/) and [NVDEC](https://developer.nvidia.com/video-codec-sdk)) solutions.```